src/RestrictedPython/transformer.py

```diff
@@ -1309,37 +1179,45 @@ def visit_While(self, node):
         """
 
         """
-        return self.generic_visit(node)
+        self.not_allowed(node)
 
     def visit_Break(self, node):
         """
 
         """
-        return self.generic_visit(node)
+        return self.node_contents_visit(node)
 
     def visit_Continue(self, node):
         """
 
         """
-        return self.generic_visit(node)
+        self.not_allowed(node)
 
-#    def visit_Try(self, node):
-#        """
-#
-#        """
-#        return self.generic_visit(node)
+    def visit_Try(self, node):
+        """Allow Try without restrictions.
 
-#    def visit_TryFinally(self, node):
-#        """
-#
-#        """
-#        return self.generic_visit(node)
+        This is Python 3 only, Python 2 uses TryExcept.
 
-#    def visit_TryExcept(self, node):
-#        """
-#
-#        """
-#        return self.generic_visit(node)
+        XXX This was forbidden in RestrictedPython 3.x maybe we have to revisit
```

Comment:
@stephan-hof Thanks for reviewing the PR. You are right. Sorry that I did not check this myself. I am going to fix it in a next PR.